bin/mkosi

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
+
+PYTHON=$(command -v python3)
+PYVERSION=$($PYTHON | cut -d ' ' -f 2 | cut -d '.' -f 1-2)
```

Comment:
Missing `--version`

Reply:
Thanks! I guess I didn't try it out again after changing that line.